stepper-2.0.md

```diff
@@ -66,17 +66,16 @@ Protocol
 
 **Stepper step (relative move)**
 
-Position is specified as a 32-bit signed long.
+Steps to move is specified as a 32-bit signed long.
 
 The speed value is a float composed of a 23-bit significand (mantissa) and a 4-bit exponent
```

Comment:
Even better if the example(s) align to a specific duration (such as 1 step per hour for "slow" and 10 steps per second for "fast"). You probably have a better sense for practical examples than I do.